backend/src/test/java/dev/tripdraw/application/TripServiceTest.java

```diff
@@ -81,4 +83,18 @@ void setUp() {
                 .isInstanceOf(TripException.class)
                 .hasMessage(TRIP_NOT_FOUND.getMessage());
     }
+
+    @Test
+    void trip_id로_여행을_조회한다() {
```

Comment:
테스트 메서드 명에서, 전반적으로 객체의 이름을 한글로 표현했던 것 같아서요,
```suggestion
    void 여행을_ID로_조회한다() {
```
가 어떨지 제안드려봅니다.

Reply:
짚어주셔서 감사합니다 놓쳤던 부분이네요 👍

Reply:
동의합니다